pkg/daemons/control/server.go

```diff
@@ -94,6 +100,8 @@ func Server(ctx context.Context, cfg *config.Control) error {
 
 	controllerManager(cfg, runtime)
 
+	cloudControllerManager(cfg, runtime)
```

Comment:
We need some way to disable the cloud-controller-manager so that someone can run an external one.